server/export.go

```diff
@@ -105,11 +105,11 @@ func ExportCmd(appExporter types.AppExporter, defaultNodeHome string) *cobra.Com
 				return err
 			}
 
-			cmd.Println(string(sdk.MustSortJSON(encoded)))
+			fmt.Fprintln(cmd.OutOrStdout(), string(sdk.MustSortJSON(encoded)))
 			return nil
 		},
 	}
-
+	cmd.SetOutput(os.Stdout)
```

Comment:
This is what I meant (SetOutput is deprecated in favor of SetOut BTW):

```suggestion
	cmd.SetOut(cmd.OutOrStdout())
	cmd.SetErr(cmd.ErrOrStderr())
```